client/signup/steps/import-url/index.jsx

```diff
@@ -68,14 +68,19 @@ class ImportURLStepComponent extends Component {
 
 	handleSubmit = event => {
 		event.preventDefault();
+		const { flowName, stepName, translate, urlInputValue } = this.props;
 		const isValid = this.validateUrl();
 
+		this.props.recordTracksEvent( 'calypso_signup_import_url', {
```

Comment:
Can we name this `calypso_signup_import_url_submit`? Event names are supposed to be named after user actions (see PCYsg-4S2-p2).